fabric-events-lifecycle-v0/src/main/resources/fabric-events-lifecycle-v0.mixins.json

```diff
@@ -7,7 +7,8 @@
     "MixinWorld"
   ],
   "client": [
-    "MixinMinecraftClient"
+    "MixinMinecraftClient",
```

Comment:
maybe sort these mixins alphabetically, incase there's gonna be more?

Reply:
Done